packages/python/plotly/plotly/missing_ipywidgets.py

```diff
@@ -0,0 +1,15 @@
+from .basedatatypes import BaseFigure
+
+
+class FigureWidget(BaseFigure):
```

Comment:
why even bother subclassing `BaseFigure` here? :)

Reply:
It seemed safer to me for it to still pass `isinstance(fig, BaseFigure)` checks, but it probably doesn't make a difference.